go.sum

```diff
@@ -272,6 +272,8 @@ github.com/vmware/vra-sdk-go v0.2.3 h1:oepqRZapum3bfzUAo+o7DYwTp97230f7yLgPu8gBV
 github.com/vmware/vra-sdk-go v0.2.3/go.mod h1:dqUEePGPUHNYQ/p9DKZM1amqB2WP1uI9JkYvzF+auVo=
 github.com/vmware/vra-sdk-go v0.2.4 h1:gIQvcT0XeFqMNX+hurvHWtGiJAEQYopzyilCm20bryg=
 github.com/vmware/vra-sdk-go v0.2.4/go.mod h1:dqUEePGPUHNYQ/p9DKZM1amqB2WP1uI9JkYvzF+auVo=
```

Comment:
Perhaps we need to "tidy" go.sum and remove the older sdk info in a future commit.